libr/debug/p/native/linux/linux_coredump.c

```diff
@@ -310,29 +297,24 @@ static bool has_map_anonymous_content(char *buff_smaps, ut64 start_addr, ut64 en
 	return 0;
 }
 
-static bool dump_this_map(char *buff_smaps, ut64 start_addr, ut64 end_addr, bool file_backed, bool anonymous, ut8 perms, ut8 filter_flags) {
-	char *p, *pp, *ppp, *extern_tok, *flags_str;
-	char *identity = NULL;
+static bool dump_this_map(char *buff_smaps, unsigned long start_addr, unsigned long end_addr, 
```

Comment:
K&R syntax solved this in a sugar way, which doesnt reduces the number of arguments, but drops type definitions from the signature line.